python/ray/tune/commands.py

```diff
@@ -283,3 +284,16 @@ def add_note(path, filename="note.txt"):
         print("Note updated at:", filepath)
     else:
         print("Note created at:", filepath)
+
+
+def kill(tune_address, port_forward, trial_id):
```

Comment:
Should this be renamed `stop_trial`? I think it might be confusing to call it kill when it just stops a trial.